libs/utils/logWebVitals.js

```diff
@@ -76,18 +76,22 @@ function isFragmentFromMep(fragPath, mep) {
   });
 }
 
+const boolStr = (val) => (val ? 'true' : 'false');
```

Comment:
No need to adapt, as this does the trick, but for brevity we could've also used
```suggestion
const boolStr = (val) => `${!!val}`;
```